### ISSUE TITLE
Microsoft Edge error with style module delayed prop

### ISSUE THREAD

**Issue by whitecolor** [snabbdom/snabbdom#360]
If `removed` property is used in `style` in MS Edge there is an error raised:

```
stack: "TypeError: Invalid calling object at nextFrame (eval code:4:33) at setNextFrame (eval code:6:5) at updateStyle (eval code:33:21) at createElm (eval code:86:17) at createElm (eval code:91:25) at createElm (eval code:91:25) at updateChildren (eval code:212:21) at patchVnode (eval code:257:21) at patch (eval code:287:13) at f (eval code:544:33)"
```
![image](https://user-images.githubusercontent.com/736697/34696172-96cfc556-f4ef-11e7-809b-29cb18a10ac9.png)

It seems due to this line:
https://github.com/snabbdom/snabbdom/blob/master/src/modules/style.ts#L9

Seems edge doesn't like assigning of native methods to some variables and using them.

Can be fixed if to replace:
```ts
var raf = (typeof window !== 'undefined' && window.requestAnimationFrame && function (cb) {window.requestAnimationFrame(cb)}) || setTimeout;
```

**Comment by nlarche** [comment]
Hello, I've got the same issue on Edge and IE11 with snabbdom v0.7.0,  do you know if a correction is planed ? 
I can make a PR if needed.
thx

**Comment by whitecolor** [comment]
You can make a PR, and hope that it will be merged someday. =)

Snabbdom maintainers seem to be not very attentive to some issues about some bugs.

@caridy @paldepind

**Comment by caridy** [comment]
@whitecolor sorry guys, I was on a long vacation, trying to get back on track. If you can send a PR (with tests please), I can take a look at it for sure.

**Comment by whitecolor** [comment]
```ts
var raf = (typeof window !== 'undefined' && (window.requestAnimationFrame).bind(window)) || setTimeout;
```

I've made PR https://github.com/snabbdom/snabbdom/pull/409

Please merge.

**Comment by mightyiam** [comment]
Seems like this was fixed in #409.
